ayekan/kyverno/policies/require_pod_requests_limits.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-requests-limits
+  annotations:
+    policies.kyverno.io/title: Require Limits and Requests
+    policies.kyverno.io/category: Best Practices
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      As application workloads share cluster resources, it is important to limit resources
```

Comment:
Does Kyverno actively enforce the policies? Or only when a new config is applied to the cluster through admissioncontrollers?

This one could cause some havoc if applied retro-active (although I agree with the permise).

Reply:
All of the policies are in audit mode, which means no enforcement.

Reply:
LGTM then